vss-extension.json

```diff
@@ -117,42 +117,76 @@
                     "displayName": "Marketplace URL",
                     "value": "https://marketplace.visualstudio.com"
                 },
-                "authenticationSchemes": [
+              "authenticationSchemes": [
+                {
+                  "type": "ms.vss-endpoint.endpoint-auth-scheme-token",
```

Comment:
As long as we don't break compat in current build definitions, it sounds good. It should work if we maintain input IDs and default the radio to VSTS Marketplace, (but we need to confirm that). Regarding the TFS endpoint, are you also proposing a new endpoint type for that? I think we can live with the Generic one.